docs/kratos/social-signin/35_gitlab.mdx

```diff
@@ -4,8 +4,87 @@ title: Add GitLab as a social sign-in provider in Ory
 sidebar_label: GitLab
 ---
 
+```mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+```
+
 # GitLab
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add GitLab as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the GitLab logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. [Create a GitLab OAuth2 Application](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application-through-the-profile).
+5. Paste the saved **Redirect URI** from Ory into the **Redirect URI** field in the GitLab form.
+6. Enable the relevant **Scopes** for the application, by ticking the relevant boxes in the GitLab form:
+    - `read_user`
+    - `openid`
+    - `profile`
+    - `email`
+7. After clicking **Save application** on the GitLab form, you see a summary of the registered application's properties.
+
+    :::caution
+
+    Make sure to copy the **Secret** field on this summary page right away.
+    This is the only opportunity you will get to copy the secret.
+
+    :::
+
+8. Copy the **Secret** from the registered application in GitLab to the **Client Secret** field on the Ory Console.
```

Comment:
You say `After clicking **Save application** on the GitLab form (...)` but you never say to actually perform this action. 

The flow of this should be: 
Click **Save application** -> Copy secret (warning applies to this activity).

Reply:
Changed to the flow to:

7. Click **Save application** on the GitLab form, which brings you to a summary of the registered application's properties.
8. On the summary page, copy the **Secret** field.